### ISSUE TITLE
go test: include calens

### ISSUE THREAD

**Issue by jtagcat** [restic/restic#3536]
Why wait 2-3m after you create PR to get the message, when you can (CONTRIBUTING.md: running all tests) `go test ./...` locally.

**Comment by jtagcat** [comment]
Abandoning this for now. Feel free to close, low priority.

Summary of what I tried doing:

changelog/testing.go → changelog/changelog_test.go (unsure if this is the right location/name/etc):
```golang
package changelog
import (
    "testing"
    "os/exec" // not in path
    "github.com/restic/calens" // not work
)

func TestChangelog(t *testing.T) {
//    cmd := exec.Command("calens")
//    if err := cmd.Run(); err != nil {
//        t.Fatal(err)
//    }

    go run github.com/restic/calens // ..?
}
```

After the first `is a program, not an importable package`, I thought requiring it in go.mod would bring it to path for os executable. I tried, but after searching for over 10m, didn't find a way to execute another external go program without making a mess.

Probably, calens would need to be turned in to a package.
